examples/globe_vector_tile.js

```diff
@@ -0,0 +1,132 @@
+/* global itowns, document */
+/* eslint-disable */
+// # Simple Globe viewer + a vector tile layer
+
+// Define initial camera position
+var positionOnGlobe = { longitude: 2.475, latitude: 48.807, altitude: 12000000 };
+var promises = [];
+
+// `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+var viewerDiv = document.getElementById('viewerDiv');
+
+// Instanciate iTowns GlobeView*
+var view = new itowns.GlobeView(viewerDiv, positionOnGlobe);
+
+function addLayerCb(layer) {
+    return view.addLayer(layer);
+}
+
+// Define projection that we will use (taken from https://epsg.io/3946, Proj4js section)
+promises.push(itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb));
+// Add two elevation layers.
+// These will deform iTowns globe geometry to represent terrain elevation.
+promises.push(itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addLayerCb));
+promises.push(itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb));
+
+var mapboxLayers = [];
+
+var count = 0;
+// Add a vector tile layer
+itowns.Fetcher.json('https://raw.githubusercontent.com/Oslandia/postile-openmaptiles/master/style.json').then(function (style) {
+    // add one layer per layer in style.json
+    style.layers.forEach(function(layer) {
+        if (layer.type === 'fill' || layer.type === 'line') {
+            mapboxLayers.push(layer);
+        }
+    });
+
+    promises.push(view.addLayer({
+        type: 'color',
+        protocol: 'xyz',
+        id: 'MVT',
+        // eslint-disable-next-line no-template-curly-in-string
+        url: 'https://osm.oslandia.io/data/v3/${z}/${x}/${y}.pbf',
+        format: 'application/x-protobuf;type=mapbox-vector',
+        options: {
+            attribution: {
+                name: 'OpenStreetMap',
+                url: 'http://www.openstreetmap.org/',
+            },
+            zoom: {
+                min: 2,
+                max: 15,
+            },
+            opacity: 0.5,
+        },
+        updateStrategy: {
+            type: itowns.STRATEGY_DICHOTOMY,
+        },
+        style: function (properties, feature) {
```

Comment:
I'd move the mapbox-style converter to itowns core.

Reply:
I was thinking of maybe doing another PR introducing Stylizers ?

Reply:
IMHO it's parsing / translating mapbox-style format, so you could already move this in `VectorTileHelper.js`